### ISSUE TITLE
Topmost on WinForms not working

### ISSUE THREAD

**Issue by oliverdarvall** [picoe/Eto#381]
Build0357 Pre-release

**Comment by cwensley** [comment]
I'm not seeing this problem.  A simple test of `new Form { Title = "Stay on top!", Topmost = true, Size = new Size(300, 300) }` works for me as well.  Do you have a repro of the issue?

**Comment by oliverdarvall** [comment]
Hi Curtis,

I got it to work as well, but I had to set Topmost in the Shown event, 
else it would just not work.

Thanks

**Comment by cwensley** [comment]
Hm, that's odd.  Could you put together a sample that reproduces the issue for you?  Perhaps something else is going on there that I'm not doing.

**Comment by oliverdarvall** [comment]
Curtis, while I was busy putting together a project to show this issue I eventually ended up with this :
.
	public class MainForm : Form
	{
		public MainForm()
		{
			Topmost = true;
			Maximizable = false;
			BackgroundColor = new Color(1, 1, 1);

			//Shown += (sender, ea) =>
			//{
			//	//Topmost = true;
			//};
		}
	}

which still had the problem in.

I then started removing references to get the project as small as possbile and when I removed the Json references it started working again.

I then added the Json references back to verify that it was the root cause and it was still working !
So now I am a bit confused.
